content/_references/envirament-variables-available-during-component-execution.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Environment variables available during component execution
+layout: article
+section: Component Descriptor
+order: 1
+category: environment-variables
+---
+
+## Description
+
+We export a number of environment variables during each task step execution. You can use these variables in your component.
+
+## API
+  * `ELASTICIO_API_USERNAME`: The name of your user to be used for authentication with [{{site.data.tenant.name}} REST API](https://api.elastic.io/docs/#introduction).
```

Comment:
* `ELASTICIO_API_USERNAME`: The name of your user to be used for authentication with {{site.data.tenant.name}} REST API.